.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -21,13 +21,7 @@ it('import and check user information', () => {
   cy.get('[data-testid="openPartialImportModal"]').click();
   const importFile = 'cypress/fixtures/import-data.json';
   cy.fixture('users').then((user) => {
-    cy.readFile(importFile).then((obj) => {
-      obj.users[0].username = `${user.importedUser.username}.${random}`;
-      obj.users[0].email = `${random}.${user.importedUser.email}`;
-      obj.users[0].lastName = `${user.importedUser.lastName}-${random}`;
```

Comment:
You have removed the logic where imported data was randomized. This is important to ensure that, when executing the tests multiple times, we can differentiate between different runs. 

The recommended flow here is to: (1) read the file, (2) randomize the data and (3) write the file with the randomized data on each run. You can take a look at this [example](https://github.com/bitnami/charts/blob/b32e7c07f0dcc798651b65768b8454bb78d6b92e/.vib/jasperreports/cypress/cypress/integration/jasperreports_spec.js#L9-L19)

Reply:
Didn't notice that. Thanks!